File: python/0027.py
class Solution(object):
    def removeElement(self, nums, val):
        """
        :type nums: List[int]
        :type val: int
        :rtype: int
        """
        if not nums:return 0
        i=0
        for j in range(len(nums)):
            if nums[j]==val:
                continue
            else:
                nums[i]=nums[j]
                i+=1
        return i

File: rust/src/bin/0027.rs
impl Solution {
    pub fn remove_element(nums: &mut Vec<i32>, val: i32) -> i32 {
        if nums.len() == 0 {
            return 0;
        }
        let mut j = 0;
        for i in 0..nums.len() {
            if nums[i] != val {
                nums[j] = nums[i];
                j += 1;
            }
        }
        j  as i32
    }
}